src/app/pages/comment/comment.component.ts

```diff
@@ -57,14 +44,32 @@ export class CommentComponent implements OnInit {
       request_id: this.object_id,
       comment: this.commentForm.controls.comment.value
     };
-    this.requestService.postComment(request).subscribe(
+   this.requestService.postComment(request).subscribe(
       response => {
         if (response) {
-          this.message = "Successfully posted the comment";
+          this.message = "Successfully posted the comment.";
+          this.comments = [];
+          this.loadComment();
```

Comment:
This will be calling the API and will be another round trip. But you all what you need already to show. You have the commenter first and last name, and the comment itself. so it will be a matter of adding those into existing comment array than going all the way to the server to add single comment.